ccd-definition/CaseEvent.json

```diff
@@ -123,5 +123,19 @@
     "ShowSummary": "Y",
     "ShowEventNotes": "N",
     "EndButtonLabel": "Submit your response"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "WITHDRAW_CLAIM",
+    "Name": "Withdraw claim",
+    "Description": "Withdraw a claim",
+    "DisplayOrder": 9,
+    "PreConditionState(s)": "CREATED",
```

Comment:
pre condition can be any state

Reply:
ticket doesn't mention any state,  I have taken this Scenario - draft claim has been issued and case is not closed as CREATED state

Reply:
updated